### DISCUSSION TITLE
Question for support braille display in NVDA Remote

### DISCUSSION THREAD

**Discussion by KihunJang1981** [NVDARemote/NVDARemote#355] [Q&A]
Dear Sir,
 
My name is Kihun Jang work on the HIMS.
We made the BrailleSense and Braille eMotion device.
We want to NVDA remote service in the Braille device directly without PC.
So we connect the NVDAremote.com server and get the “speak” data.
But we want to receive the braille data from 
 
We send the “type: set_braille_info”, “name: Braille Sense” and “numCells: 32”
But we can’t get the braille data.
So can you let me know how to get the braille data?
Thank you.
 
Best Regards,
Kihun Jang,

**Comment by KihunJang1981** [comment]
Dear Sir,

I sucess get the braille display data.
Remain issue is not work "braille_input."
Cna you advice "braille_input"?

Thank you.

**Reply by ChrisDuffley** [reply]
Hi @ctoth

He's now asking about being able to send commands I think. I think he wants to make a 3rd-party client for the BrailleSense and Emotion but is unable to figure out what to use to send data to the remote machine. He's said that he can receive Braille data fine now.

Hope this helps!

**Comment by KihunJang1981** [comment]
Dear @ctoth,

Can you help the braille_input function?

Best Regards,
Kihun Jang,
